native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -157,6 +169,26 @@ const BpkBadge = (props: Props) => {
 
   return (
     <View style={viewStyle}>
+      {icons &&
+        icons.map((icon, index) => (
+          <Fragment key={`${icon}`}>
+            <BpkIcon
+              small
+              icon={icon}
+              style={JSON.parse(JSON.stringify(textStyle))}
```

Comment:
For some reason, this fixes the flow issue I was seeing.
Don't know why - this isn't really a solution!